tutorials/plot_artifacts_correction_ica.py

```diff
@@ -56,14 +56,24 @@
 #              explicitly set ``fir_design='firwin'`` to use this method. This
 #              is the recommended filter method for ICA preprocessing.
 
+
 ###############################################################################
 # Fit ICA
 # -------
 #
-# ICA parameters:
+# First, choose the ICA method. There are currently 4 possible choices:
+# `fastica`, `picard`, `infomax` and `extended-infomax`.
+#
+# .. note:: The default method in MNE is FastICA, which along with Infomax is
+#           one of the most widely used ICA algorithm. Picard is a
+#           new algorithm that is expected to converge faster than FastICA and
+#           Infomax, especially when the aim is to recover accurate maps with
+#           a low tolerance parameter.
```

Comment:
"... parameter, see [1]_ for more information." and add `References` section at the bottom with your paper. Even a preprint is okay for now, we can update it later.